pysqa/queueadapter.py

```diff
@@ -407,7 +407,12 @@ def set_queue_adapter(
         # The RemoteQueueAdapter has additional dependencies, namely paramiko and tqdm.
         # By moving the import to this line it only fails when the user specifies the
         # RemoteQueueAdapter in their pysqa configuration.
-        from pysqa.base.remote import RemoteQueueAdapter
+        try:
+            from pysqa.base.remote import RemoteQueueAdapter
+        except ImportError as e:
+            raise ImportError(
+                "Failed to instantiate RemoteQueue setup, probably due to missing optional dependencies"
+            ) from e
```

Comment:
I am a bit reluctant to add this try except statement, as the message does not add a lot more details.

Reply:
I was only confused, since I installed pure pyiron_base to test some things and got greeted by an ImportError. I immediately checked if our feedstock is missing packages and then realized it is an optional dependency. 
This is only conveyed by the code base. 
However, I am fine with not having this in.